tests/foreman/cli/test_import.py

```diff
@@ -151,6 +167,63 @@ def import_user_data():
     ],)
 
 
+def import_hc_data():
```

Comment:
The name of this function does not seems to be paired with the docstring. What hc means?